.github/workflows/gene-essentiality.yml

```diff
@@ -0,0 +1,32 @@
+name: Check gene essentiality with Hart 2015
+
+on:
+  pull_request:
+    branches:
+      - "main"
+
+jobs:
+  check-metabolictasks:
+    runs-on: self-hosted
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run gene essentiality
+        id: essentiality
+        run: |
+          TEST_RESULTS=$(/usr/local/bin/matlab -nodisplay -nosplash -nodesktop -r "ihuman = importYaml('Human-GEM.yml'); taskStruct = parseTaskList('data/metabolicTasks/metabolicTasks_Essential.txt'); eGenes = estimateEssentialGenes(ihuman, 'Hart2015_RNAseq.txt', taskStruct); disp(evaluateHart2015Essentiality(eGenes));" | awk 'NR>9 && !/^\.+/')
```

Comment:
@haowang-bioinfo the output of this line is very hard to capture/use. Do you have any suggestions to make this output more compact?

Reply:
As seen [in this workflow run](https://github.com/SysBioChalmers/Human-GEM/actions/runs/6429997344/job/17460119226?pr=675), the output is way too rich to fit in a PR comment. @haowang-bioinfo how would you suggest to change it to end up with a nice summary like [here](https://github.com/SysBioChalmers/Human-GEM/pull/700#issuecomment-1738614979)?

Reply:
no, I've never thought about this. Because they are non-binary indicators that are hard to be used for evaluating purpose in actions